volpe/builder_utils.py

```diff
@@ -3,12 +3,12 @@
 from llvmlite import ir
 
 from tree import TypeTree, volpe_assert, get_obj_key_value
-from volpe_types import unwrap, is_int, is_flt, char
+from volpe_types import unwrap, is_int, is_flt, is_char
 
 
 def math(self, tree: TypeTree):
     values = self.visit_children(tree)
-    if is_int(tree.return_type):
+    if is_int(tree.return_type) or is_char:
```

Comment:
this is missing some arguments

Reply:
Wow, I am surprised that worked. I guess it just treated is as true.

Reply:
that's python for yah